Extension/src/LanguageServer/client.ts

```diff
@@ -2660,6 +2661,23 @@ export class DefaultClient implements Client {
         }
     }
 
+    public async handleCheckForCompiler(): Promise<void> {
+        await this.awaitUntilLanguageClientReady();
+        const compilers: configs.KnownCompiler[] | undefined = await this.getKnownCompilers();
+        if (!compilers || compilers.length === 0) {
+            const compilerName: string = process.platform === "win32" ? "MSVC" : (process.platform === "darwin" ? "Clang" : "GCC");
+            vscode.window.showInformationMessage(localize("no.compilers.found", "No C++ compilers were found on your system. Install {0} using the instructions in the editor.", compilerName), { modal: true });
+        } else {
+            const header: string = localize("compilers.found", "We found the following C++ compiler(s) on your system:");
```

Comment:
This list will contain C and C++ compilers, i.e. gcc and g++ -- is that intentional or did we want to filter out the C versions here?

Reply:
Intentional